calico/calcollections.py

```diff
@@ -75,3 +75,89 @@ def resulting_size(self):
     @property
     def empty(self):
         return not (self.added_entries or self.removed_entries)
+
+
+class OneToManyIndex(object):
```

Comment:
Renamed to MultiValueDict; I think that's a more standard name for this datastructure.